website/static/js/pages/discussions-page.js

```diff
@@ -0,0 +1,16 @@
+/** Initialization code for the project discussion page. */
+var $ = require('jquery');
+var Comment = require('../comment.js');
+
+// Initialize comment pane w/ it's viewmodel
+var userName = window.contextVars.currentUser.name;
+var canComment = window.contextVars.currentUser.canComment;
+var hasChildren = window.contextVars.node.hasChildren;
+var target = window.contextVars.comment_target;
+target = (target==='None')? 'node' : target;
```

Comment:
Target should never be `'None'`; are you correctly jsonifying `comment_target`?

Why assign `target` then immediately reassign it? Perhaps use a different variable name for the intermediate result.